assets/admin/students/student-modal/course-list.js

```diff
@@ -117,15 +123,14 @@ export const CourseList = ( { searchQuery, onChange } ) => {
 					'/wp/v2/courses?per_page=100' +
 					( query ? `&search=${ query }` : '' ),
 				method: 'GET',
+				signal: getSignal(),
 			} )
-				.then( ( result ) => {
-					setCourses( result );
-				} )
-				.catch( () => {
-					setIsFetching( false );
-				} )
+				.then( ( result ) => setCourses( result ) )
+				.catch( () => {} )
```

Comment:
Long comment without an action item: 😂


This line `.catch( () => {} )`  is silencing all errors incoming from the `api-fetch`.
If you remove it and run the tests, you will see a lot of `You are probably offline.`

Investigating a bit more a found that the `api-fetch` version we are using[ hides the abort errors.](https://github.com/WordPress/gutenberg/blob/%40wordpress/api-fetch%403.22.0/packages/api-fetch/src/index.js#L102)

New `api-fetch` versions [are exposing errors incoming from aborted requests.
](https://github.com/WordPress/gutenberg/blob/%40wordpress/api-fetch%403.22.0/packages/api-fetch/src/index.js)

I think, considering our context, we should ignore all abort errors and re-throw the other ones, but considering today we haven't a way to identify them until we update the `api fetch`.

I don't have a suggestion for this case, all scenarios seem not so ideal for me, but I think it is the best we can do.

Reply:
> If you remove it and run the tests, you will see a lot of You are probably offline.

This is the exact reason I put it back in actually. 😆

What do you think about [logging the error to the console](https://github.com/Automattic/sensei/pull/5065/commits/5ec3bb86e086eda348f536944bfdec33fa5ddb10) for now? This will give Happiness Engineers some extra info when troubleshooting potential issues with fetching courses. The tests will be a bit noisier, but I think that's OK given the extra info.

Reply:
Yes, I think the log will help us to remember that we need to solve it properly.

I still don't know if something is wrong with our test setup that is causing this error or if the `api-fetch` really should throw an expectation, and is our responsibility to filter this error.

Reply:
According to the official specs, we should just filter the AbortError
https://dom.spec.whatwg.org/#aborting-ongoing-activities-example
So probably we should do it after updating the api-fetch.